allennlp/training/metrics/perplexity.py

```diff
@@ -0,0 +1,46 @@
+from overrides import overrides
+
+from allennlp.training.metrics.metric import Metric
+
+import math
+
+@Metric.register("ppl")
```

Comment:
characters are cheap, is there a good reason not to register it as `"perplexity"`?